test/index/types/types.snapshot.cc

```diff
@@ -45,15 +45,15 @@
     class X {};
   
     namespace {
-//  ^^^^^^^^^ definition [..] $ANON/types.cc/
+//  ^^^^^^^^^ definition [..] a/`$anonymous_namespace_types.cc`/
```

Comment:
This bit is technically a bug-fix, since the previous scheme would've led to the same symbol for anonymous namespaces in different contexts in the same file.

```cpp
namespace a { namespace { } }
namespace b { namespace { } }
```